tests/test_cmdline.py

```diff
@@ -652,16 +637,12 @@ def test_check_unused__simple_project__reports_only_unused(
     )
 
     expect = [
+        "",
         "These dependencies appear to be unused (i.e. not imported):",
         "- 'pandas' declared in:",
         f"    {tmp_path / 'requirements.txt'}",
     ]
-    expect_logs = [
-        f"INFO:fawltydeps.extract_imports:Parsing Python files under {tmp_path}",
-        "INFO:fawltydeps.packages:'pandas' was not resolved."
-        " Assuming it can be imported as 'pandas'.",
-    ]
-    output, errors, returncode = run_fawltydeps(
+    output, returncode = run_fawltydeps_function(
```

Comment:
I suspect the extra empty line is needed because `run_fawltydeps()` performs a `.strip()` on the captured output where `run_fawltydeps_function()` does not?

Reply:
Yes! :bowing_woman: 
Adding `.strip()` to the latter.